### DISCUSSION TITLE
What maxcll and max fall value to enter in StaxRip

### DISCUSSION THREAD

**Discussion by Victory61** [staxrip/staxrip#1565] [Q&A]
Good morning.
I am struggling with a doubt that I can't solve, based on the HDR10 plotting obtained I can't figure out what are the correct values to enter in the encoder (x265).
For MaxFall I could enter the average value for MaxCll I don't know whether to enter the values given at the top of the table by percentage or the Maximum value.
Thank you 

![02- GROSSO GUAIO A CHINATOWN (1986) 4K BT2020 HDR_plot](https://github.com/user-attachments/assets/d33ea52e-fa43-4443-a162-569fcf661ec6)

**Comment by Dendraspis** [comment]
Good morning,
why don't you just keep the values from the source file?

**Comment by Victory61** [comment]
Merry Christmas
The source file is in SDR, this is my own HDR classification via DaVinci whose file exported to BT2020 ProRes was plotted via DoVi script, so I am undecided about the values of MaxFall and MaxCLL.

**Reply by Dendraspis** [reply]
In that case you have to test it a bit and/or find a way to find good default values. Nothing I can help you with ...

**Comment by Victory61** [comment]
Thank you anyway.
